nextflow_schema.json

```diff
@@ -132,14 +132,14 @@
                 },
                 "seqwish_transclose_batch": {
                     "type": "string",
-                    "default": "10M",
+                    "default": "10000000",
                     "description": "Number of base pairs to use for transitive closure batch.",
                     "help_text": "If you run out of memory during the seqwish step, you can lower this value. It will take longer, but it will use less memory.",
                     "pattern": "^([1-9]\\d*[kKmMgGtT]?|0)$"
                 },
                 "seqwish_sparse_factor": {
                     "type": "number",
-                    "default": 0.0,
+                    "default": 0,
```

Comment:
Please keep the 0.0 here @heringerp. It can indeed be a number and I want to indicate that.